ui/src/app/utils/stores.tsx

```diff
@@ -141,8 +141,18 @@ export interface ServerConfigStore {
 
 export const serverConfigStore = atom<ServerConfigStore>({});
 
-export interface CanComponentDeactivate {
-  canDeactivate: () => Promise<boolean> | boolean;
+export interface MatchParams {
```

Comment:
Correct, they need to match what's declared in the URL. Will comment. Can even get fancy w/test too 🙂

Reply:
Ok, it looks like there's not a way to do this at runtime because TypeScript compiles to Javascript, types aren't real, everything is made up, and the points don't matter